README.md

```diff
@@ -73,3 +81,103 @@ optional arguments:
                         associated matrix conditions, a total of Nx2 builds
                         will be identified. (default: 2)
 ```
+
+
+Example
+=======
+
+Supposing we have two moderately complex conda recipes in a directory:
+
+```
+$ mkdir -p my_recipes/recipe_a my_recipes/recipe_b
+$ cat <<EOF > my_recipes/recipe_a/meta.yaml
+package:
+  name: recipe_a
+  version: 2.4
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+EOF
+
+$ cat <<EOF > my_recipes/recipe_b/meta.yaml
+package:
+  name: recipe_b
+  version: 3.2
+
+requirements:
+  build:
+    - recipe_a
+    - numpy x.x
+  run:
+    - recipe_a
+    - python
+    - numpy x.x
+
+EOF
+```
+
+If we wish to build the lot, we can simply run:
+
+```
+$ conda-build-all my_recipes
+
+conda-build-all my_recipes
+Fetching package metadata: ........
+Resolving distributions from 2 recipes... 
+Computed that there are 11 distributions from the 2 recipes:
+Resolved dependencies, will be built in the following order: 
+    recipe_a-2.4-py26_0 (will be built: True)
+    recipe_a-2.4-py27_0 (will be built: True)
+    recipe_a-2.4-py34_0 (will be built: True)
+    recipe_a-2.4-py35_0 (will be built: True)
+    recipe_b-3.2-np19py26_0 (will be built: True)
+    recipe_b-3.2-np110py27_0 (will be built: True)
+    recipe_b-3.2-np19py27_0 (will be built: True)
+    recipe_b-3.2-np110py34_0 (will be built: True)
+    recipe_b-3.2-np19py34_0 (will be built: True)
+    recipe_b-3.2-np110py35_0 (will be built: True)
+    recipe_b-3.2-np19py35_0 (will be built: True)
+
+BUILD START: recipe_a-2.4-py26_0
+...
+
+```
+
+As you can see, these two unassuming recipes will result in more than 2 builds.
+In this case, ``recipe_a`` has been identified to be built against the top two minor versions of the top two major versions of Python - that is, py26, py27, py34, py35 (at the time of writing).
+Next, ``recipe_b`` has been identified to be built against the top two minor versions, of the top two major versions of Python *and* numpy. If all built distributions of python and numpy were available, there would be ``4 x 2`` permutations (4 being the number of Python versions available, and 2 being the number of numpy versions, assuming there exists only 1 major version of numpy, otherwise this would double to 4).
+
+We've seen that we can build a *lot* of distributions for our simple recipes. We can tighten the build matrix somewhat by adding or own conditions:
+
+```
+$ conda-build-all my_recipes --matrix-condition "python 3.5.*" "numpy >=1.8" --matrix-max-n-minor-versions=0
+Fetching package metadata: ........
+Resolving distributions from 2 recipes... 
+Computed that there are 4 distributions from the 2 recipes:
+Resolved dependencies, will be built in the following order: 
+    recipe_a-2.4-py34_0 (will be built: True)
```

Comment:
You added `python 3.5.*` to the matrix conditions yet this still shows builds against 3.4. Do you mean to set the matrix condition as `python 3.*` instead?